### ISSUE TITLE
Web Page not available

### ISSUE THREAD

**Issue by tklose** [yu210148/gvm_install#92]
The install appeared to go smoothly. After reboots the web page is not running. 
Linux 5.15.0-1020-azure x86_64.  ubuntu 22_04 LTS Gen 2. 

Where do I look for the issue?

**gvmd.log** 
md manage:WARNING:2022-10-03 17h44.31 utc:5660: Failed to open lock file '/run/gvmd/gvm-checking': No such file or directory
md   main:CRITICAL:2022-10-03 17h44.31 utc:5660: gvmd: Error trying to get checking lock

**Tried these commands:**
gvm-check-setup          command not found
openvas-check-setup     command not found

**Comment by yu210148** [comment]
Humm, that's different. I'm not sure what's going on there and I'm in the
middle of moving so I can't troubleshoot at the moment. Has anyone else
seen this?

Kev.

**Comment by darnellkeithj** [comment]
Yes, just happened to me. gvm-check-setup doesn't work as he said. Everything loaded and worked just fine until a reboot of the system.
